src/containers/Tenant/Schema/SchemaTree/SchemaTree.tsx

```diff
@@ -33,15 +33,26 @@ export function SchemaTree(props: SchemaTreeProps) {
     const [schemaTreeKey, setSchemaTreeKey] = React.useState('');
 
     const fetchPath = async (path: string) => {
-        const promise = dispatch(
-            schemaApi.endpoints.getSchema.initiate({path}, {forceRefetch: true}),
-        );
-        const {data} = await promise;
-        promise.unsubscribe();
-        if (!data?.[path]) {
+        let schemaData: TEvDescribeSchemeResult | undefined;
+        do {
+            const promise = dispatch(
+                schemaApi.endpoints.getSchema.initiate({path}, {forceRefetch: true}),
+            );
+            const {data, originalArgs} = await promise;
+            promise.unsubscribe();
+            // Check if the result from the current request is received. rtk-query may skip the current request and
+            // return data from a parallel request, due to the same cache key.
+            if (originalArgs?.path === path) {
+                schemaData = data?.[path];
+                break;
+            }
+            // eslint-disable-next-line no-constant-condition
+        } while (true);
```

Comment:
Why to do this in `do while` cycle? `await` is not enough?

Reply:
As described in the comment, the "promise" may return result from another request (with another params), so we try to repeat the request in this case.